itk_zlib.h

```diff
@@ -23,9 +23,9 @@
 /* Use the zlib library configured for ITK.  */
 #include "itkThirdParty.h"
 #ifdef ITK_USE_SYSTEM_ZLIB
-# include <zlib.h>
+#  include <zlib.h>
```

Comment:
I am not sure whether ITK style should be applied to MetaIO's zlib copy.

Reply:
Please do not format third party code. It makes reconciling with upstream changes that much harder.